custom/covid/management/commands/add_hq_user_id_to_case.py

```diff
@@ -30,8 +31,12 @@ def update_cases(self, domain, case_type, user_id):
         skip_count = 0
         for case in accessor.iter_cases(case_ids):
             username_of_associated_mobile_workers = case.get_case_property('username')
-            user_id_of_mobile_worker = username_to_user_id(normalize_username(username_of_associated_mobile_workers,
-                                                                              domain))
+            try:
+                user_id_of_mobile_worker = username_to_user_id(normalize_username(
```

Comment:
Nitpick: it's good to make the footprint of a `try` block as small as possible, because it's time-consuming when refactoring to figure out which parts trigger the exception(s). For this case, that would mean just calling `normalize_username` in assigning that to a variable to use after the try/except.

Reply:
Yeah that's very fair: [d63bdc3](https://github.com/dimagi/commcare-hq/pull/29163/commits/d63bdc3f524e6ea5a1a2425082acabf4c265785d)